src/games/strategy/triplea/ai/proAI/data/ProTerritoryManager.java

```diff
@@ -1103,7 +1103,7 @@ private void findBombardOptions(final PlayerID player, final List<Territory> myU
       combinedUnits.addAll(patd.getMaxAmphibUnits());
       ProLogger.debug("Removing territory that we can't successfully attack: " + t + ", maxWin%="
           + patd.getMaxBattleResult().getWinPercentage() + ", maxAttackers=" + combinedUnits.size());
-      result.remove(t);
+      result.remove(attackMap.get(t));
```

Comment:
Re-use this variable: `final ProTerritory patd = attackMap.get(t);`? 
If so, would also be good to give the variable a more meaningful name as well.